app/assets/stylesheets/show.scss

```diff
@@ -0,0 +1,6 @@
+@import "foundation_and_overrides";
+$black: #000000;
+
+body {
+  color: $black;
+}
```

Comment:
Changing the body font color is the kind of thing that it's better to do in `foundations_and_overrides`. Look for a commented-out line in that file that seems to describe what you're trying to change, then uncomment it and change the value! 

Also it's a little counterintuitive to have this file named `show.scss`, since the styles that you wrote here are unspecific enough that they'll show up on all pages, not just a show page.